src/app/shared/services/layout.service.ts

```diff
@@ -8,17 +8,17 @@ import { delay } from 'rxjs/operators';
 export class LayoutService {
   // Service allowing modifications of the layout
 
-  private leftMenuAndHeadersDisplayed = new BehaviorSubject<boolean>(true);
-  leftMenuAndHeadersDisplayed$ = this.leftMenuAndHeadersDisplayed.asObservable().pipe(delay(0));
+  private fullFrameContent = new BehaviorSubject<boolean>(true);
+  fullFrameContent$ = this.fullFrameContent.asObservable().pipe(delay(0));
 
-  private withTask = new BehaviorSubject<boolean>(false);
-  withTask$ = this.withTask.asObservable().pipe(delay(0));
+  private footer = new BehaviorSubject<boolean>(false);
```

Comment:
contentFooter maybe?
And I assume "footer false" should mean that the footer is not shown. So as you changed withTask -> footer, I would expect you invert the logic (so the default value).  Or you have to call the attribute `noContentFooter`, which is valid as well (double negation is not great though)

Reply:
I had changed the logic to fit the new name "footer" (if true, adds the footer margin). I just renamed it to contentFooter which is a better name.

This comment made me realize that I didn't update the logic when renaming `leftMenuAndHeaders` to `fullFrameContent`, so I just fixed that.